debian/preinst

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh -e
 
 case "$1" in
-    configure)
-    adduser --system --group --no-create-home --quiet plight || true
+    configure|install)
```

Comment:
How did you build this on other nodes?

I had to add "install" as an option for preinst